packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -311,7 +313,9 @@ export class FASTTooltip extends FASTElement {
         this.addEventListener("mouseout", this.mouseoutAnchorHandler);
         this.addEventListener("mouseover", this.mouseoverAnchorHandler);
 
-        document.addEventListener("keydown", this.keydownDocumentHandler);
+        if (!("CloseWatcher" in window)) {
+            document.addEventListener("keydown", this.keydownDocumentHandler);
+        }
```

Comment:
Are there any issues with having both a `closeWatcher` and a `keydown` event handler? How might the two methods collide?

Reply:
I'm unaware of what the result would be if you had both. I imagine it'd be okay?

Reply:
Worse case I see is that we run the "close the thing" function twice which shouldn't break regardless, but I assume closeWatcher marks the event with preventDefault or stops propagation so we can avoid acting twice?